--- python2-orig/setup.py	2016-04-13 15:19:08.857977000 +0200
+++ python2-patch/setup.py	2016-04-13 17:43:40.338838216 +0200
@@ -1020,12 +1020,14 @@
             missing.append('_bsddb')
 
         # The sqlite interface
-        sqlite_setup_debug = False   # verbose debug prints from this script?
+        sqlite_setup_debug = True   # verbose debug prints from this script?
 
         # We hunt for #define SQLITE_VERSION "n.n.n"
         # We need to find >= sqlite version 3.0.8
-        sqlite_incdir = sqlite_libdir = None
-        sqlite_inc_paths = [ '/usr/include',
+        sqlite_incdir = 'SQLITE_RECIPE_INC'
+        sqlite_libdir = 'SQLITE_RECIPE_LIB'
+        sqlite_inc_paths = [ sqlite_incdir,
+                             '/usr/include',
                              '/usr/include/sqlite',
                              '/usr/include/sqlite3',
                              '/usr/local/include',
@@ -1071,7 +1073,7 @@
                     print "sqlite: %s had no SQLITE_VERSION"%(f,)
 
         if sqlite_incdir:
-            sqlite_dirs_to_check = [
+            sqlite_dirs_to_check = [ sqlite_libdir,
                 os.path.join(sqlite_incdir, '..', 'lib64'),
                 os.path.join(sqlite_incdir, '..', 'lib'),
                 os.path.join(sqlite_incdir, '..', '..', 'lib64'),
@@ -1083,6 +1085,8 @@
                 sqlite_libdir = [os.path.abspath(os.path.dirname(sqlite_libfile))]
 
         if sqlite_incdir and sqlite_libdir:
+            if sqlite_setup_debug:
+                print "sqlite_libdir: %s"%(sqlite_libdir,)
             sqlite_srcs = ['_sqlite/cache.c',
                 '_sqlite/connection.c',
                 '_sqlite/cursor.c',
